improver/nbhood/nbhood.py

```diff
@@ -293,17 +318,18 @@ def __init__(
                 originally masked.
         """
         super(NeighbourhoodProcessing, self).__init__(
-            neighbourhood_method, radii, lead_times=lead_times)
+            neighbourhood_method, radii, lead_times=lead_times
+        )
 
-        methods = {
-            "circular": CircularNeighbourhood,
-            "square": SquareNeighbourhood}
+        methods = {"circular": CircularNeighbourhood, "square": SquareNeighbourhood}
         try:
             method = methods[neighbourhood_method]
-            self.neighbourhood_method = method(
-                weighted_mode, sum_or_fraction, re_mask)
+            self.neighbourhood_method = method(weighted_mode, sum_or_fraction, re_mask)
         except KeyError:
-            msg = ("The neighbourhood_method requested: {} is not a "
-                   "supported method. Please choose from: {}".format(
-                       neighbourhood_method, methods.keys()))
+            msg = (
+                "The neighbourhood_method requested: {} is not a "
+                "supported method. Please choose from: {}".format(
+                    neighbourhood_method, methods.keys()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [dict.keys referenced when not iterating](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5385606)